pkg/upgrade/drain.go

```diff
@@ -121,12 +120,7 @@ func evictPods(ctx context.Context, kc *kubernetes.Clientset, nodeName string) e
 			continue
 		}
 
-		err = kc.CoreV1().Pods(pod.Namespace).Evict(&policy.Eviction{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      pod.Name,
-				Namespace: pod.Namespace,
-			},
-		})
+		err = kc.CoreV1().Pods(pod.Namespace).Delete(pod.Name, &metav1.DeleteOptions{})
```

Comment:
maybe worth renaming the `evictPods` function too

Reply:
done